composer.json

```diff
@@ -17,13 +17,13 @@
     }
   ],
   "require": {
-    "php": ">=7.0",
-    "monolog/monolog": "^1.23",
-    "ext-curl": "*"
+    "php": "^7.0",
+    "ext-curl": "*",
+    "ext-json": "*",
```

Comment:
Hmm IIRC, json_encode is only used in test, so we should move it to require-dev

Reply:
+1